crates/bevy_scene/src/lib.rs

```diff
@@ -40,9 +41,8 @@ impl Plugin for ScenePlugin {
             .add_asset::<Scene>()
             .init_asset_loader::<SceneLoader>()
             .init_resource::<SceneSpawner>()
-            .add_systems(Update, scene_spawner_system)
             // Systems `*_bundle_spawner` must run before `scene_spawner_system`
-            .add_systems(PreUpdate, scene_spawner);
+            .add_systems(PreUpdate, (scene_spawner, scene_spawner_system).chain());
```

Comment:
I played with it locally and I think `PostUpdate` will have more sense because it will also remove one frame delay for spawning scenes in `Update`. Sorry for the mess. I updated the original issue.
```suggestion
            .add_systems(PostUpdate, (scene_spawner, scene_spawner_system).chain());
```